wpeverest/everest-forms
# Fix - Block design all messed up when viewed in block editor.

### All Submissions:


### Changes proposed in this Pull Request:

Previously, The design of the form was messed in the block editor when we choose a form. This PR fixes that design issue.
Note :- Please before test run **grunt css**. No need to build.
![image](https://github.com/wpeverest/everest-forms/assets/109399375/019aa77c-b75e-4ebe-abf9-c2751e7faf06)

Please test the zip in the two or more testing site. Zip link:- https://themegrill.atlassian.net/browse/EVF-1026?focusedCommentId=21855


### How to test the changes in this Pull Request:

1. Drag a form selector block and choose block
2. Verify the design of that form in block editor
3.Also verify the form's front-end validation

### Types of changes:


### Other information:



### Changelog entry

> Fix - Block design all messed up when viewed in block editor.
